tests/integration/src/query_service/a_ttl.py

```diff
@@ -0,0 +1,466 @@
+"""
+Summary:
+This test file contains integration tests for Time-To-Live (TTL) and custom retention policies in SigNoz's query service.
+It verifies the correct behavior of TTL settings for traces, metrics, and logs, including support for cold storage, custom retention conditions, error handling for invalid configurations, and retrieval of TTL settings.
+"""
+
+import json
+import time
+from http import HTTPStatus
+from typing import Dict, Any
+
+import pytest
+import requests
+
+from fixtures import types
+from fixtures.auth import USER_ADMIN_EMAIL, USER_ADMIN_PASSWORD
+from fixtures.logger import setup_logger
+from fixtures.logs import insert_logs, Logs
+import threading
+
+logger = setup_logger(__name__)
+
+
+class TestTTLMethods:
```

Comment:
```suggestion
class TestTTLMethods:
```

Please also run `make py-fmt` and `make py-lint` at the root of the repository.